app/gimlet-rot/app-b.toml

```diff
@@ -11,6 +11,7 @@ version = 0
 [kernel]
 name = "gimlet-rot"
 requires = {flash = 32768, ram = 4096}
+features = []
```

Comment:
This doesn't look necessary? If it is feel free to disregard.

Reply:
This is very unfortunate. Once I added a new section for `sram3` to `tasks.update_server`, it appeared that I had to add this line everywhere to get the containing images to build.

Reply:
Turns out this is unnecessary. I just needed to bump humility locally.